cardinal/system/player_spawner.go

```diff
@@ -5,22 +5,23 @@ import (
 
 	"pkg.world.dev/world-engine/cardinal"
 	"pkg.world.dev/world-engine/cardinal/message"
+	"pkg.world.dev/world-engine/cardinal/types"
 
 	comp "github.com/argus-labs/starter-game-template/cardinal/component"
 	"github.com/argus-labs/starter-game-template/cardinal/msg"
 )
 
+const (
+	InitialHP = 100
+)
+
 // PlayerSpawnerSystem spawns players based on `CreatePlayer` transactions.
 // This provides an example of a system that creates a new entity.
 func PlayerSpawnerSystem(world cardinal.WorldContext) error {
 	return cardinal.EachMessage[msg.CreatePlayerMsg, msg.CreatePlayerResult](
 		world,
 		func(create message.TxData[msg.CreatePlayerMsg]) (msg.CreatePlayerResult, error) {
-			maxHp := 100
-			id, err := cardinal.Create(world,
-				comp.Player{Nickname: create.Msg.Nickname},
-				comp.Health{HP: maxHp},
-			)
+			id, err := createPlayer(world, create.Msg.Nickname)
```

Comment:
Abstractions are not desirable for the starter game template because they hide away the code that we want to highlight.

Reply:
removed